### PR TITLE
update dev seed

### PR DESCRIPTION
**Pull request by HaGuesto**

### CODE REVIEW

**Comment by HaGuesto** [comment]
[2.0 add source_size_id and target_size_id to shipment details](https://trello.com/c/fUFfZEOP/1195-20-add-sourcesizeid-and-targetsizeid-to-shipment-details)

**Comment by HaGuesto** [comment]
[2.0 Enable clearly showing shipment details of completed shipment](https://trello.com/c/AhtWx3LO/1183-20-enable-clearly-showing-shipment-details-of-completed-shipment)

**Comment by HaGuesto** [comment]
[2.0 Change box state transitions on shipment target side](https://trello.com/c/GEcg1ymw/1184-20-change-box-state-transitions-on-shipment-target-side)

File: back/init.sql [hunk 1]
```diff
@@ -1592,15 +1600,23 @@ CREATE TABLE `shipment_detail` (
   KEY `shipment_detail_source_location_id` (`source_location_id`),
   KEY `shipment_detail_target_location_id` (`target_location_id`),
   KEY `shipment_detail_created_by_id` (`created_by_id`),
-  KEY `shipment_detail_deleted_by_id` (`deleted_by_id`),
+  KEY `shipment_detail_deleted_by_id` (`removed_by_id`),
```
Comment (pylipp) [inline back/init.sql:1603]: The naming is not matching anymore here, but I don't think it's an issue @HaGuesto

**Review by pylipp — APPROVED** [review]

**Pull request merged**